### PR TITLE
Receive user input from frontend

### PR DESCRIPTION
**Pull request by jennygaoo**
Now, the user can input their simulation params and receive the zip code summary table. I realized that my backend wasn't actually accounting for when the user wants to request all zip codes (zip code parameter is left blank), so I fixed that. Here are some manual tests:

in the console, I've logged all of the user inputs and at the bottom, the corresponding JSON is logged. 
Screenshot #1 is when the user enters a single zip code. Screenshot #2 is when the user doesn't enter a specific zip code (so they request all zip codes). The error messages are unrelated (bc the maps aren't set up yet)

Screenshot 2020-07-17 at 10 37 34 AM
Screenshot 2020-07-17 at 10 37 51 AM

### CODE REVIEW

File: postal-injection/src/main/java/com/google/sps/common/GetZipCodeSummary.java [hunk 1]
```diff
@@ -45,14 +45,13 @@
...
-  }
-
   public static HashMap<String, ZipCodeData> getProcessedZipCodeData(String userSpecifiedZipCode) {
     ZipCodeDataProcessor zipCodeDataProcessor = new ZipCodeDataProcessor();
-    return zipCodeDataProcessor.getZipcodeData(userSpecifiedZipCode);
+    if (userSpecifiedZipCode.isEmpty()) {
+      return zipCodeDataProcessor.getZipcodeData();
+    } else {
+      return zipCodeDataProcessor.getZipcodeData(userSpecifiedZipCode);
+    }
```
Comment (jgao23) [inline postal-injection/src/main/java/com/google/sps/common/GetZipCodeSummary.java:54]: supernit: don't need else statement:

```
if (userSpecifiedZipCode.isEmpty()) {
  return zipCodeDataProcessor.getZipcodeData();
}

return zipCodeDataProcessor.getZipcodeData(userSpecifiedZipCode);
```

File: postal-injection/src/main/webapp/summary_table.js [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
       postOfficeToCustomerRate);
 }
 
+function getUserInputs() {
+  const zipCode = document.getElementById('zip-code').value;
+  const regularRate = document.getElementById('regular-rate').value;
+  const palletRate = document.getElementById('pallet-rate').value;
+  const minPalletSize = document.getElementById('min-pallet-size').value;
+  const threshold = document.getElementById('threshold').value;
+  const postOfficeToCustomerRate =
```
Comment (jgao23) [inline postal-injection/src/main/webapp/summary_table.js:18]: in a different PR, i would highly recommend doing validation on the inputs

File: postal-injection/src/main/java/com/google/sps/common/GetZipCodeSummary.java (hunk 1 above)
Comment (jgao23) [reply postal-injection/src/main/java/com/google/sps/common/GetZipCodeSummary.java:54]: also, can't remember if you have a test for this class but if you do please add a test

File: postal-injection/src/main/java/com/google/sps/common/GetZipCodeSummary.java [hunk 3]
```diff
@@ -45,14 +45,13 @@
   }
 
   /** gets the ZipCodeData objects using ZipCodeDataProcessor */
-  public static HashMap<String, ZipCodeData> getProcessedZipCodeData() {
-    ZipCodeDataProcessor zipCodeDataProcessor = new ZipCodeDataProcessor();
-    return zipCodeDataProcessor.getZipcodeData();
```
Comment (k-sharpe) [inline postal-injection/src/main/java/com/google/sps/common/GetZipCodeSummary.java:50]: The ZipCodeDataProcessor is a static class, you don't need to create a new ZipCodeDataProcessor you can just do ZipCodeDataProsessor.getZipcodeData();

File: postal-injection/src/main/java/com/google/sps/common/GetZipCodeSummary.java (hunk 1 above)
Comment (jennygaoo) [reply postal-injection/src/main/java/com/google/sps/common/GetZipCodeSummary.java]: I'll add screenshots of manual tests below!

**Comment by jennygaoo** [comment]
image
image

**Pull request merged**